packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -234,11 +234,11 @@ const ruleset: any = {
 
     // RPC Code: RPC-Policy-V1-05
     AvoidAdditionalProperties: {
-      description: "The use of additionalProperties is not allowed except for user defined tags on tracked resources.",
+      description: "Definitions must not have properties named additionalProperties except for user defined tags or predefined references.",
       severity: "error",
       stagingOnly: true,
       message: "{{description}}",
-      resolved: true,
+      resolved: false,
```

Comment:
>false [](http://example.com/codeflow?start=16&length=5)

This fix is incorrect. There could be models referencing other non common types and those need to be resolved. The fix should be to not apply the logic just for managedidentities

Reply:
changed back and addressed delegate and userAssignedIdentity Resources